comparadise-utils/screenshots.ts

```diff
@@ -21,15 +21,27 @@ export function baseExists(path: string) {
   return exists;
 }
 
+export type CompareScreenshotArgs = {
+  screenshotFolder: string;
+  pixelMatchOptions?: PixelMatchOptions;
+};
+
 /**
  * Runs a visual regression test.
- * @param screenshotFolder - Full screenshots folder where the base/new/diff
- *                           images will be compared and written to.
+ * @param args -
+ * Contains Full screenshots folder where the base/new/diff images will be compared and written to
+ * Optionally contains specified Pixelmatch config settings.
  */
-export function compareScreenshots(screenshotFolder: string) {
+export function compareScreenshots(args: CompareScreenshotArgs) {
+  const { screenshotFolder, pixelMatchOptions } = args || {};
```

Comment:
```suggestion
  const { screenshotFolder, pixelMatchOptions } = args;
```
We know `args` must be defined